examples/green_screen/README.md

```diff
@@ -45,11 +46,9 @@ This example requires OpenCV to be installed to build. To ensure it will be buil
 #### Windows
 
 Our recommended way of getting OpenCV on Windows is by installing pre-built libraries. There's even a PowerShell script
-that'll do much of the work for you in `scripts/install-opencv.ps1`. This will place OpenCV in your `C:\` folder. You'll
-also need to add the folder where the OpenCV DLLs are to your PATH. If you used the script, that means you need to add
-`C:\opencv\build\x64\vc14\bin` to your PATH. If you installed OpenCV somewhere else, you may need to add the
-`-DOpenCV_DIR=path\to\a\folder\in\your\opencv\install\that\contains\the\file\OpenCVConfig.cmake` option to cmake when
-you build.
+that'll do much of the work for you in `scripts/install-opencv.ps1`. This will place OpenCV in your `C:\` folder. CMake
+will copy the OpenCV binaries to the the `${CMAKE_BINARY_DIR}` folder so that all built binaries will run as expected. A 
```

Comment:
```suggestion
will copy the OpenCV binaries to the the `${CMAKE_RUNTIME_OUTPUT_DIRECTORY}` folder so that all built binaries will run as expected. A 
```